nipype/interfaces/mrtrix3/preprocess.py

```diff
@@ -241,6 +242,151 @@ def _format_arg(self, name, trait_spec, value):
                 return f"-{trait_spec.argstr}"
         return super()._format_arg(name, trait_spec, value)
 
+
+class DWIPreprocInputSpec(MRTrix3BaseInputSpec):
+    in_file = File(
+        exists=True, argstr="%s", position=-10, mandatory=True, desc="input DWI image"
```

Comment:
Are these `position` arguments really necessary for all inputs?

Reply:
Not necessary, this was my lazy way of ensuring the interface command stayed consistent with the examples from the mrtrix documentation. 

I think if position is left empty, the inputs appear right after `dwifslpreproc`? I was going for a format of `dwifslpreproc in_file out_file -rpe_options -other_options" to follow the examples [here](https://mrtrix.readthedocs.io/en/latest/reference/commands/dwifslpreproc.html#dwifslpreproc). 

Is there a way to achieve that without specifying positions for everything?

Reply:
If that's necessary, I think you could set position of 0 for `in_file`, 1 for `out_file`, 2 for `rpe_options`, and leave it unset for the rest. If it's not actually necessary for an option, I would personally leave it unset, because you're indicating to future maintainers that they need to be very particular about this.